jjcollinge/servicefabric
# Use Go coding style.

- Use `httptest` instead of a mock
- interfaces are implicit, only need to create by clients of the libs
- build URL in only one method
- check `nil` to avoid panicing
- fix `continueToken` in `GetProperties`
- use `tls.Config`
- add Travis configuration
- add Makefile
- add readme
